### PR TITLE
PEER20-274 Hide Language switch 

### PR DESCRIPTION
**Pull request by alexeishulga**
commented out language selection

### CODE REVIEW

File: app/containers/ChangeLocale/index.js [hunk 1]
```diff
@@ -49,41 +49,43 @@ export const ChangeLocale = ({ locale, changeLocaleDispatch, withTitle }) => {
...
-        </ul>
-      }
-      id="choose-language-dropdown"
-      isArrowed
-      isMenuLabelMobile
-      isArrowMarginMobile
-    />
+    <>
+    </>
+    // <Dropdown
```
Comment (ruslanzharkov) [inline app/containers/ChangeLocale/index.js:54]: could you please add TODO with ticket number, because it's not obvious why this code is commented

Comment (alexeishulga) [reply app/containers/ChangeLocale/index.js]: I made a mistake in the logic of the task.
I had to remove the language switcher but not the whole component.
On the call they said to do it temporarily.
Now everything is fixed.

**Review by ruslanzharkov — APPROVED** [review]

File: app/containers/ChangeLocale/index.js (hunk 1 above)
Comment (ruslanzharkov) [reply app/containers/ChangeLocale/index.js]: good, checked it, looks fine

Comment (alexeishulga) [reply app/containers/ChangeLocale/index.js]: thanks

**Pull request merged**